components/common/CommonInputImage.vue

```diff
@@ -98,7 +98,9 @@ watch(file, (image, _, onCleanup) => {
       bg="black/30" text-white
       flex justify-center items-center
     >
-      <div class="i-ri:loader-4-line animate-spin animate-duration-[2.5s]" text-4xl />
+      <div class="animate-spin animate-duration-[2.5s] preserve-3d">
```

Comment:
this is also not allowed

Reply:
same for divs inside buttons:

![imagen](https://user-images.githubusercontent.com/6311119/212356170-6d14d38b-f12a-4fb3-be49-0866816dab9b.png)

Reply:
quite right. those should also be converted to spans.

Reply:
fixing label...

Reply:
done